plugin/targetd/targetd.py

```diff
@@ -148,6 +152,20 @@ def plugin_register(self, uri, password, timeout, flags=0):
         self.headers = {'Content-Type': 'application/json',
                         'Authorization': 'Basic %s' % (auth,)}
 
+        if "no_ssl_verify" in self.uri["parameters"] \
+                and self.uri["parameters"]["no_ssl_verify"] == 'yes':
+            self.no_ssl_verify = True
+
+        if "cert_file" in self.uri["parameters"]:
+            # Check for file existence and throw error now if not present
+            self.cert_file = self.uri["parameters"]["cert_file"]
+
+            if not (os.path.exists(self.cert_file) and
+                    os.path.isfile(self.cert_file)):
```

Comment:
`os.path.isfile` also check the existence: https://docs.python.org/2/library/os.path.html#os.path.isfile